FrontEnd/composables/useAuth.js

```diff
@@ -1,47 +1,58 @@
 // composables/useAuth.js
-// import { useAuthStore } from '@/store/auth';
-import { useUserStore } from "~/stores/useUserStore";
-import { useNuxtApp } from "#app";
-import { srvLogin, srvSignup } from "~/services/authServices";
+import { useUserStore } from "~/stores/user";
+import { useCookie } from '#app';
+
 
 export const useAuth = () => {
-  const { $api } = useNuxtApp();
-  //   const authStore = useAuthStore();
+  const { $customApi } = useNuxtApp();
+  const { $toast } = useNuxtApp();
   const userStore = useUserStore();
 
-  const handleAuthError = (error) => {
-    console.error("Authentication error: ", error.message);
-  };
 
   const login = async (credentials) => {
     try {
-        const {data} = await srvLogin($api, credentials);
-        console.log("response after fetching data : ", data);
-          // localStorage.setItem('token', data.token);
-          // localStorage.setItem('userInfo', JSON.stringify(data.user) );
-          userStore.setUserInfo(data.user);
-          userStore.setToken(data.token);
-          navigateTo("/")
+        console.log("custom api : ",$customApi)
+        const response = await $customApi.post('/auth/login', credentials);
+
+        console.log("response after fetching data : ", response);
+         return response.data
     } catch (error) {
-      handleAuthError(error);
-      // Error handling is already done in the interceptor
+      console.log("error for the auth fetch : ",error.message);
+        $toast({
+          text: error?.message || 'Login failed', 
+          style: {
+            background: "rgba(255, 0, 0, 0.7)", 
+            borderRadius: "8px", 
+            color: "white",
+          }
```

Comment:
define error styles in the plugin and pass params what type of error needs to be used.

example $toast.success(), $toast.error()

Reply:
Updated toast plugin as suggested. Please review the latest commit